README.md

```diff
@@ -30,19 +91,45 @@ Simulariumio converts simulation outputs to the format consumed by the [Simulari
 * Conversions for data from custom engines can be implemented using the TrajectoryConverter class
 * Also accepts metrics data for plots to display alongside spatial data
 
-We're working to improve performance for converting large trajectories, and also discussing with the authors of some packages the possibility of adding the ability to export Simularium files directly.
+We're discussing the possibility of adding the ability to export Simularium files directly with the authors of some packages.
 
-___
+Current rendering capabilities of the viewer:
+* Spheres: by default, each agent in a scene is represented as a single sphere
+* Mesh surfaces: represent each agent as a mesh files, e.g. coarse molecular surfaces
+* Multi-sphere: provide Protein Databank .pdb files or PDB IDs
+* Line representations for fibers, filaments, or bonds
 
-## Visualize results
-1. In a supported browser (Firefox, Chrome, or Edge), navigate to https://simularium.allencell.org/viewer.
-2. Drag the file output from SimulariumIO from your file browser onto the window or use the file upload dialogue to choose your file
+Rendering capabilities planned for future:
+* Volume rendering for RDME or PDE-based simulation results
+* Support for .cif files and coarse-grain sphereTree files for multi-sphere rendering
+
+<br/>
 
-## Sharing links to results
-1. Upload your Simularium file to one of the supported public cloud providers, currently Dropbox, Google Drive, or Amazon S3, and get a publicly accessible link to the file.
-2. In a supported browser, navigate to https://simularium.allencell.org/viewer?trajUrl=[link to your file]. You can share this link with collaborators or post it on your website so that others can interactively view your results.
+---
 
-___
+<br/>
+
+## Installation
+**Install Requires:** 
+
+* Requires Python 3.7 or 3.8
```

Comment:
just curious, is 3.9 supported?

Reply:
I think it would work but I need to test, there's an issue for it #55